example.diff

```diff
@@ -1,3 +1,52 @@
+--- a/document.txt
```

Comment:
Can you add the new examples at the bottom of the file ? This impacts the tests in a way that hides the impact of the whole PR.